clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -123,10 +123,16 @@ New checks
   Gives warnings for tagged unions, where the number of tags is
   different from the number of data members inside the union.
 
+- New :doc:`modernize-use-integer-sign-comparison
+  <clang-tidy/checks/modernize/use-integer-sign-comparison>` check.
+
+  Replace comparisons between signed and unsigned integers with their safe
+  ``std::cmp_*`` alternative.
```

Comment:
Actually, please add 

> C++20 ``std::cmp_*`` alternative, if available.

Since the replacement is only available since C++20.

(+ synchronize with documentation and heaeder comment)